### PR TITLE
Paginate AJAX

### PR DESCRIPTION
**Pull request by thanhsangcao**

### CODE REVIEW

**Comment by thanhsangcao** [comment]
ready

**Comment by thanhsangcao** [comment]
ready

**Comment by thanhsangcao** [comment]
ready

File: resources/views/admin/service/load.blade.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        @foreach ($services as $service)
+            <tr>
+                <td>{{ $service->id }}</td>
+                <td><a href="{{ asset('admin/services/' . $service->id . '/edit') }}">{{ $service->name }}</a></td>
+                <td>{{ $service->price }}</td>
+                <td>
+                    <a href="{{ asset('admin/services/' . $service->id . '/edit') }}" class="btn btn-warning"><i class="fa fa-pencil" aria-hidden="true"></i> {{ __('Edit') }}</a>
+                    {{ Form::open(array( 'url' => 'admin/services/' . $service->id, 'class' => 'pull-right')) }}
+                        {{ Form::hidden('_method', 'DELETE') }}
+                        {{ Form::submit('Delete', array('class' => 'btn btn-danger','onClick' => "return confirm('Are you sure ? ')")) }}
```
Comment (BunnyPi04) [inline resources/views/admin/service/load.blade.php]: convention

File: resources/views/admin/user/load.blade.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                <td>{{ $user->email }}</td>
+                <td>{{ $user->address }}</td>
+                <td>{{ $user->phone_number }}</td>
+                <td>{{ $user->permission }}</td>
+                <th>{{ $user->salon->name }}</th>
+                <td>
+                    <a href="{{ asset('admin/users/' . $user->id . '/edit') }}" class="btn btn-warning"><i class="fa fa-pencil" aria-hidden="true"></i> {{ __('Edit') }}</a>
+                    {{ Form::open(array('url' => 'admin/users/' .  $user->id, 'class' => 'pull-right')) }}
+                        {{ Form::hidden('_method', 'DELETE') }}
+                        {{ Form::submit('Delete', array('class' => 'btn btn-danger','onClick' => "return confirm('Bạn có chắc chắn muốn xóa')")) }}
```
Comment (BunnyPi04) [inline resources/views/admin/user/load.blade.php]: convention
i18n

**Comment by BunnyPi04** [comment]
commented

**Comment by thanhsangcao** [comment]
ready

**Pull request merged**